packages/kbn-reporting/export_types/printable_pdf_v2/lib/generate_pdf.ts

```diff
@@ -1,20 +1,23 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
  */
 
 import * as Rx from 'rxjs';
 import { mergeMap, tap } from 'rxjs/operators';
-import { PdfScreenshotResult } from '@kbn/screenshotting-plugin/server';
-import { TaskPayloadPDFV2 } from '../../../../common/types/export_types/printable_pdf_v2';
-import { ReportingServerInfo } from '../../../core';
-import { ReportingConfigType } from '../../../config';
-import type { LocatorParams, PdfMetrics, UrlOrUrlLocatorTuple } from '../../../../common/types';
-import type { PdfScreenshotOptions } from '../../../types';
-import { getFullRedirectAppUrl } from '../../common/v2/get_full_redirect_app_url';
-import { getTracker } from '../../common/pdf_tracker';
+import type { PdfScreenshotOptions, PdfScreenshotResult } from '@kbn/screenshotting-plugin/server';
+import {
+  getFullRedirectAppUrl,
+  getTracker,
+  ReportingServerInfo,
+  TaskPayloadPDFV2,
+} from '@kbn/reporting-export-types-helpers-server';
+import type { LocatorParams, ReportingConfigType } from '@kbn/reporting-common';
+import { PdfMetrics } from '@kbn/reporting-common/metrics';
+import { UrlOrUrlWithContext } from '@kbn/screenshotting-plugin/server/screenshots';
 
 interface PdfResult {
```

Comment:
Looks like this file also exists in `packages/kbn-reporting/export_types_helpers_server/generate_pdf.ts`